`strip-ansi-escapes` new release `v0.1.2` is breaking reedline.

**Platform**  macOS, 
**Terminal software** zsh

With the new strip-ansi-escapes release https://github.com/luser/strip-ansi-escapes/issues/15 , reedline is not compiling in my project.
 
## Steps to reproduce

1. `Cargo.toml`:

```toml
[dependencies]
reedline = "0.22.0"
```

2. I'm getting the following error when compiling reedline on a MacOS M2:
```
error[E0599]: no method named `map_err` found for struct `Vec<u8>` in the current scope
  
39 | /     strip_ansi_escapes::strip(string)
40 | |         .map_err(|_| ())
   | |         -^^^^^^^ method not found in `Vec<u8>`
   | |_________|
   | 
```

Am I doing something wrong? Thanks :)

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -23,7 +23,7 @@ nu-ansi-term = "0.49.0"
 rusqlite = { version = "0.29.0", optional = true }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = { version = "1.0.79", optional = true }
-strip-ansi-escapes = "0.1.1"
+strip-ansi-escapes = "0.1.2"
 strum = "0.25"
 strum_macros = "0.25"
 thiserror = "1.0.31"

--- a/src/painting/utils.rs
+++ b/src/painting/utils.rs
@@ -36,9 +36,8 @@ pub(crate) fn coerce_crlf(input: &str) -> Cow<str> {
 ///
 /// If parsing fails silently returns the input string
 pub(crate) fn strip_ansi(string: &str) -> String {
-    strip_ansi_escapes::strip(string)
+    String::from_utf8(strip_ansi_escapes::strip(string))
         .map_err(|_| ())
-        .and_then(|x| String::from_utf8(x).map_err(|_| ()))
         .unwrap_or_else(|_| string.to_owned())
 }
 

